src/hostapi/wasapi/pa_win_wasapi.c

```diff
@@ -306,6 +306,14 @@ typedef struct _pa_AudioClientProperties {
     #define PA_THREAD_ID DWORD
 #endif
 
+
+//Visual Studio 2010 does not like the inline keyword
+#if _MSC_VER <= 1600
+    #define PA_INLINE _inline
+#else
+    #define PA_INLINE inline
+#endif
```

Comment:
I propose to modify this whole block to:

```c
#if (_MSC_VER <= 1600)
    #define inline _inline
#endif
```

and move it inside the compatibility block `#if defined(_MSC_VER) && (_MSC_VER >= 1400)` right after the line https://github.com/PortAudio/portaudio/blob/8ccd53fe7ced184c67fcd5d22939a2cf383b4597/src/hostapi/wasapi/pa_win_wasapi.c#L89

And as a result please undo all changes where inline was replaced with PA_INLINE.

It will be much cleaner compatibility fix without an intrusion to a code which is compliant with modern compilers.

Reply:
agreed and done